blog/2025/fixes-enabled-dnfts-expected.md

```diff
@@ -0,0 +1,64 @@
+---
+category: 2025
+date: "2025-06-03"
+template: '../../@theme/templates/blogpost'
+seo:
+    description: 
+labels:
+    - Amendments
+markdown:
+    editPage:
+        hide: true
+---
+# Two Fix Amendments Enabled, dNFTs Expected
+
+Two fixes to the XRP Ledger protocol, [fixFrozenLPTokenTransfer](/resources/known-amendments#fixfrozenlptokentransfer) and [fixInvalidTxFlags](/resources/known-amendments#fixinvalidtxflags), became enabled on 2025-015-15. Additionally, the [DynamicNFT amendment](/resources/known-amendments#dynamicnft) has gained support from a supermajority of validators and will become enabled on 2025-06-11 if it maintains continuous support. The minimum core server version to support these amendments is **v2.4.0**; any servers on older versions are now amendment blocked.
```

Comment:
2025-05-15?

Also, I'm always a little gun-shy about predicting the future, even with a caveat.

Reply:
Yep, 2025-05-15. Fixed.

As for the prediction, I'm with you, but comfortable with this one because it's just describing the behavior of the ledger (it has support, it has a scheduled time, and it'll happen if it maintains support).